src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null,
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleKeydown);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.handleKeydown);
+  }
+
+  handleKeydown = (event:KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    const key = this.state.pressedKey;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          {(key === null
+            && 'Nothing was pressed yet')
+            || `The last pressed key is [${key}]`}
```

Comment:
```suggestion
          {(!key
            ? 'Nothing was pressed yet')
            : `The last pressed key is [${key}]`}
```

it's better to use ternary for such cases